### PR TITLE
Club/News/ Fetch

### PR DESCRIPTION
**Pull request by GabrielAdward**

### CODE REVIEW

**Comment by GabrielAdward** [comment]
@albinforsbergju Skulle du kunna ladda ned den och kolla om din frontend uppdateras, min uppdateras ej

**Comment by albinforsbergju** [comment]
Hittade problemet. 
```
let n = fetch(`http://localhost:8080/club/${id}/news`)
    n.then((res) => {
        return res.json()
    }).then((data) => {
        news = data.news
    })
```

Det ska vara `news = data`. Kolla in dokumentationen i postman!

File: backend/src/routes/club.js [hunk 1]
```diff
@@ -37,24 +38,111 @@ router.get("/:id", async (req, res) => {
...
+    `, [req.params.id, req.params.newsId]);
+    res.json(news);
+  } catch (err) {
+    res.status(500).json({ message: err.message });
+  }
+});
+
+  //create news 
+  
+  router.post("/:id/news", async (req, res) => {
```
Comment (albinforsbergju) [inline backend/src/routes/club.js:86]: This should be in the teams route? A team creates news, not the club. The club shows all the team news with teams connected to the club.

File: backend/src/routes/club.js [hunk 2]
```diff
@@ -37,24 +38,111 @@ router.get("/:id", async (req, res) => {
...
+    const { title, content, datetime, teamId } = req.body;
```
Comment (albinforsbergju) [inline backend/src/routes/club.js:87]: Not `datetime`, should be `published`.
`
Should also include `PersonId`

**Review by albinforsbergju — COMMENTED** [review]
Will update with more comments when I have time.

File: backend/src/routes/club.js (hunk 2 above)
Comment (GabrielAdward) [reply backend/src/routes/club.js:87]: Then I should only have get request displaying all news and specific news

Comment (albinforsbergju) [reply backend/src/routes/club.js:87]: Should be in teams also.

Comment (GabrielAdward) [reply backend/src/routes/club.js:87]: So I should also delete the specific news page?

**Comment by GabrielAdward** [comment]
@albinforsbergju Jag kör en PR på denna så att den kan merga med Main som innehåller förhoppningsvis Calendar

**Review by albinforsbergju — APPROVED** [review]
LGTM!

**Pull request merged**